x-pack/test/rule_registry/common/lib/helpers/cleanup_target_indices.ts

```diff
@@ -17,7 +17,7 @@ export const cleanupTargetIndices = async (getService: GetService, user: User, s
     const aliasMap = await es.indices.getAlias({
       name: targetIndices,
       allow_no_indices: true,
-      expand_wildcards: 'open',
```

Comment:
I think this was the main reason the rule registry test below was skipped.  That test was the best place for uuid tests, so managed to get it running again \o/